x-pack/plugins/observability_solution/apm/server/routes/entities/get_entities.ts

```diff
@@ -65,6 +68,7 @@ export async function getEntities({
               ...environmentQuery(environment, SERVICE_ENVIRONMENT),
               ...entitiesRangeQuery(start, end),
               ...termQuery(ENTITY_TYPE, EntityType.SERVICE),
+              ...(serviceName ? termQuery(SERVICE_NAME, serviceName) : []),
```

Comment:
is the check needed? I think termQuery handles that

Reply:
you're right!